src/validators/custom.js

```diff
@@ -0,0 +1,19 @@
+function custom(value, options) {
```

Comment:
What you think about use customValidations as name of that? because just custom don't say much about the function in my vision...